source/first-steps/install/getting-meshroom/platforms/osx.rst

```diff
@@ -1,7 +1,7 @@
-macOS
+MacOS
```

Comment:
This guide is for the 2019.1 release. I don´t know if it works for the 2020 release

Reply:
https://github.com/alicevision/meshroom/wiki/MacOS